t/0001.xml

```diff
@@ -237,7 +237,7 @@
 <t id="1044">Scatter Gun</t>
  <t id="1041">Scatter Gun</t>
  <t id="1045">Scatter Gun</t>
- <t id="1042">Size: SMALL\n Weapon Class: LIGHT\n A shotgun weapon used in close-range fights. Can be a great damage-dealer if used in experienced hands.</t>

```

Comment:
- is an operator and cannot be used with spaces
not really needed but added for testing if the operator is an issue with this text, you may manually undo this if you see fit.

Reply:
Please see commit #5ce2db1
for correction to _Arange_ typo made by me in this commit.